src/config-import.js

```diff
@@ -193,6 +199,14 @@ export async function readConfig(configPath) {
             presetValue[key] = preset[key]
           }
         }
+        if ('icon' in preset && typeof preset.icon === 'string') {
+          if (!iconFilenames.has(preset.icon)) {
+            throw new Error(
+              `preset references icon with name ${preset.icon} but file doesn't exist`
+            )
+          }
+        }
```

Comment:
This has two problems:

1. It throws an error, rather than adding a warning.
2. `preset.icon` is required, but this doesn't check that.

Fixed this in #791.